clients/webassembly/js-example/index.js

```diff
@@ -24,10 +24,16 @@ async function main() {
     // sets up better stack traces in case of in-rust panics
     set_panic_hook();
 
-    // validator server we will use to get topology from
-    const validator = "http://testnet-validator1.nymtech.net:8081"; //"http://localhost:8081";
+    // validator we will use to get topology from
+    // If you run into CORS errors make sure to set `enable-unsafe-cors` as true
+    // and add `'Access-Control-Allow-Origin'` to the accepted headers
+    // in your validator's `config.toml`,
+    const validator = "http://mixnet.club:1317"; //  1317 = standard API port.
```

Comment:
you should probably be using different validator - this one is not running milhon

Reply:
this way of getting the topology is now out of date but changed it anyway - should we handle the new example in a different pull?

Reply:
I think it's reasonable to fix it in this pull.